docs/en/stack/ml/nlp/ml-nlp-e5.asciidoc

```diff
@@ -175,9 +175,9 @@ https://ml-models.elastic.co/multilingual-e5-small.vocab.json
 
 For the optimized version, you need the following files in your system:
 ```
-https://ml-models.elastic.co/multilingual-e5-small-optimized.metadata.json
-https://ml-models.elastic.co/multilingual-e5-small-optimized.pt
-https://ml-models.elastic.co/multilingual-e5-small-optimized.vocab.json
+https://ml-models.elastic.co/multilingual-e5-small-linux-x86-64.metadata.json
+https://ml-models.elastic.co/multilingual-e5-small-linux-x86-64.pt
+https://ml-models.elastic.co/multilingual-e5-small-linux-x86-64.vocab.json
```

Comment:
```suggestion
https://ml-models.elastic.co/multilingual-e5-small_linux-x86_64.metadata.json
https://ml-models.elastic.co/multilingual-e5-small_linux-x86_64.pt
https://ml-models.elastic.co/multilingual-e5-small_linux-x86_64.vocab.json
```

Reply:
A very confusing mix of underscores and dashes

Reply:
I checked the links for ELSER and they are correct

https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-elser.html#elser-model-artifacts